### ISSUE TITLE
Every WordPress page load calls remote server, transient cache is not used correctly

### ISSUE THREAD

**Issue by ottok** [digitoimistodude/dude-facebook-feed#3]
On Seravo's server we noticed that sites using this plugin failed to load immediately if their outgoing network connection was cut:
```
# wp-test-remote-failure 
Testing baseline for https://porvoonpooki.fi ...
=> Got response status 200 with length 19636
Testing if https://porvoonpooki.fi continues to work without remote connections...
=> Got response status 200 with length 13813
==> Result changed, site most likely failed to work!
=> Got response status 200 with length 13813
==> Result changed, site most likely failed to work!
=> Got response status 200 with length 19636
=> Got response status 200 with length 13813
==> Result changed, site most likely failed to work!
=> Got response status 200 with length 13813
==> Result changed, site most likely failed to work!
=> Got response status 200 with length 19636
=> Got response status 200 with length 13813
==> Result changed, site most likely failed to work!
=> Got response status 200 with length 13813
==> Result changed, site most likely failed to work!
=> Got response status 200 with length 19636

Restoring remote connections...
Errors detected! Please check manually for details.
```

So clearly something is making remote calls during the page load. There is no need for that to happen on every page load..

**Comment by ottok** [comment]
Seems this was already fixed in 6802076dfd2f01226080583ddf7495e1d59a4d77 in May. Please publish version 1.0.2 to get this fix out.

**Comment by ottok** [comment]
The same fix should also go into:
- https://github.com/digitoimistodude/dude-insta-feed/blob/master/dude-insta-feed.php#L113
- https://github.com/digitoimistodude/dude-twitter-feed/blob/master/dude-twitter-feed.php#L171

And probably others.

**Comment by ottok** [comment]
We see this on a lot of sites with our tool presented in slide 34 at https://www.slideshare.net/ottokekalainen/the-5-most-common-reasons-for-a-slow-wordpress-site-and-how-to-fix-them-extended-edition#34

Please release a new version as this is fixed already in https://github.com/digitoimistodude/dude-facebook-feed/commit/6802076dfd2f01226080583ddf7495e1d59a4d77 in May by a colleague to @TeemuSuoranta

**Comment by timiwahalahti** [comment]
@ottok Thanks for reporting this and sorry for a slight delay with addressing the issue!

6802076 has been merged to master in May, which is essentially always the most current version and should be used. We don't have separate releases for these plugins (tho we should have, but that's a different story and something I'll adress when these plugins will go under total rewrite at some point).

I updated the plugin version number in master to `0.1.1`. Also added a throttiling to API calls when we don't get a proper response. Before we did make a new call on every page load if previous one failed. Now new calls will be blocked for some time. Other plugins affected got these updates also.

Again, thanks Otto for reporting this!
